coldfront/plugins/qumulo/templates/email/invalid_users.txt

```diff
@@ -0,0 +1,9 @@
+Notice for the following allocation:
+{{allocation_url}}
+
+The following users were note found and could not be granted {{ access_type }} access:
+{% for user in invalid_users %}
+{{ user }}
+{% endfor %}
+
+Please inspect and correct as needed.  The allocation will still be created or updated as normal.
```

Comment:
It might be worth rephrasing this to something like - "The allocation was created or updated as normal with those users that could be found."

I know we want to avoid confusing messages, but I think this is a place where context is helpful.